### ISSUE TITLE
Unexpected output in STDERR when -echo4 is used

### ISSUE THREAD

**Issue by maxpozdeev** [exiftool/exiftool#72]
Hi! First of all, thank you for this tool and updates!

After updating to new version of ExifTool I has unexpected results. When I use ExifTool with -stayopen I pass -echo4={ready}. I expect to receive "{ready}\n" in STDERR, but I see "{ready}\n\n" - one more newLine char. 

Seems the code changed in v12.10 is broken and need to be fixed.

old:
```
@echo3 and print STDOUT join("\n", @echo3), "\n";
@echo4 and print STDERR join("\n", @echo4), "\n";
```

new:
```
if (@echo4) {
    my $str = join "\n", @echo4, "\n";
    $str =~ s/\$\{status\}/$rtnVal/ig;
    print STDERR $str;
}
```

**Comment by boardhead** [comment]
Oops.  Wow.  That's a dumb mistake I made there.  Sorry.  I'll fix this in 12.15.

- Phil

**Comment by maxpozdeev** [comment]
Thank you
